frontend/packages/console-shared/src/components/dashboard/status-card/OperatorStatusBody.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { GetOperatorsWithStatuses, LazyLoader } from '@console/plugin-sdk';
+import { AsyncComponent, FirehoseResourcesResult } from '@console/internal/components/utils';
+import { HealthState } from './states';
+import { getMostImportantStatuses } from './state-utils';
+
+import './operator-body.scss';
+
+export const OperatorsSection: React.FC<OperatorsSectionProps> = ({
+  resources,
+  getOperatorsWithStatuses,
+  title,
+  linkTo,
+  rowLoader,
+}) => {
+  const error = _.values(resources).some((r) => r.loadError);
+  const operatorStatuses = getOperatorsWithStatuses(resources);
+  const sortedOperatorStatuses = getMostImportantStatuses(operatorStatuses)
+    .sort((a, b) => a.operators[0].metadata.name.localeCompare(b.operators[0].metadata.name))
+    .slice(0, 5);
```

Comment:
yes, max 5 is by design

Reply:
Hm, thinking about this again, maybe we should reconsider and show all degraded/failed operators  in the popover rather than just the first alphabetical 5 (which is arbitrary). I think that would be more useful.

Reply:
Ok, limit is now removed